docs/WritingTests.asciidoc

```diff
@@ -672,8 +672,8 @@ eventually workers can be held back for the cluster.
 It is possible to ensure that all jobs within the same _parallel_ cluster are
 executed on the same worker host. This is useful for connecting the SUTs without
 having to connect the physical worker hosts. Use `PARALLEL_ONE_HOST_ONLY=1` to
-enable this. Note that adding this setting in `workers.ini` has currently *no*
-effect.
+enable this. This setting can be applied as a test variable during the time
+of scheduling as well as in the worker configuration file `workers.ini`.
```

Comment:
The whole feature hasn't been implemented yet so adding the documentation now is a bit problematic (as it would not really work just yet).

Reply:
We won't merge this PR in this stage but I consider the approach to write the documentation first :)

Reply:
With my latest changes we can include this documentation.